tests/eth_getBalance/get-balance-blockhash.io

```diff
@@ -1,2 +1,2 @@
->> {"jsonrpc":"2.0","id":9,"method":"eth_getBalance","params":["0xaa00000000000000000000000000000000000000","0x2ce0e9a8a0b33d45aeb70cf6878d2943deddcf6600e06b84546eaf0a0d2b9643"]}
-<< {"jsonrpc":"2.0","id":9,"result":"0x1"}
+>> {"jsonrpc":"2.0","id":1,"method":"eth_getBalance","params":["0xaa00000000000000000000000000000000000000","0x0c5761302ec071b5c04130bf2c8c9167fe8f2efeb756c47d0f44eff683e1203e"]}
```

Comment:
This actually helped me catch the schema error from #326 because it has a leading 0 which isn't allowed for uint, but is for hash32.